### DISCUSSION TITLE
Unable to fetch Intellij License for Developer pack

### DISCUSSION THREAD

**Discussion by shruthi-mat** [community/community#113644] [GitHub Education]
### Select Topic Area

Question

### Body

Hi Team,

I am unable to enable the license for my Github developer Pack.
It gives the error no active licenses are associated with the account.

Please guide on the issue.

Thanks
Shruthi

**Comment by Amandacdev** [comment]
Hello, Shruthi

The first thing you should do is check if your email on IntelliJ account is the same as your GitHub email. Sometimes, IntelliJ uses a different email from the main email registered on GitHub, and that can cause some problems. 

Also, check your validation email! Maybe your license just needs to be activated. You can try to activate the license manually in IntelliJ. Go to "Help" -> "Register" in IntelliJ and try to insert your license key manually. 

Finally, you can try to contact IntelliJ Support. With the information you provided, I think that's how I can help you.

Good luck! :D

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @shruthi-mat ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
